src/components/business-lookup/BusinessLookup.vue

```diff
@@ -111,11 +132,17 @@ export default class BusinessLookup extends Vue {
   protected searchResults: Array<BusinessLookupResultIF> = []
   protected selectedBusiness: BusinessLookupResultIF = null
 
-  /** The text field validation rules. */
+  /** The business lookup validation rules. */
   readonly businessLookupRules: Array<(v) => boolean | string> = [
     v => !!v || 'Business is required'
   ]
 
+  /** The business name validation rules. */
+  readonly businessNameRules: Array<(v) => boolean | string> = [
+    (v: string) => !!v?.trim() || 'Business or corporation name is required',
+    (v: string) => (v?.length <= 150) || 'Cannot exceed 150 characters' // maximum character count
+  ]
```

Comment:
Same rules as in Edit UI.

Ref: https://github.com/bcgov/business-edit-ui/blob/a3a1416a82d14de2c299e21f04ec18e8ca4a5a1b/src/components/common/PeopleAndRoles/OrgPerson.vue#L835